src/integrationTest/java/com/mongodb/kafka/connect/embedded/EmbeddedKafka.java

```diff
@@ -225,7 +225,7 @@ private Properties connectWorkerConfig() {
         workerProps.put("key.converter.schemas.enable", "false");
         workerProps.put(DistributedConfig.VALUE_CONVERTER_CLASS_CONFIG, "org.apache.kafka.connect.storage.StringConverter");
         workerProps.put("value.converter.schemas.enable", "false");
-        workerProps.put(DistributedConfig.OFFSET_COMMIT_INTERVAL_MS_CONFIG, "30000");
+        workerProps.put(DistributedConfig.OFFSET_COMMIT_INTERVAL_MS_CONFIG, "100");
```

Comment:
This configuration change showed the bug.  The offset (where a user is in a topic) is now updated every 100ms.  This means when the cursor is invalidated (via a drop) the code path tries to create a new change stream using the saved resume token.

Previously, it wouldn't get a resume token (as none had been saved) and then it would start a new change stream cursor with no `resumeAfter`.

Reply:
Since this is an interval, does it make the test racy?  Or is it guaranteed to have the effect that you want?

Reply:
Not with how the tests are written, they are designed to ensure that events eventually appear on a topic - so the test polls.  This ensures that the offset is written by kafka which the connector uses.  The theoretical issue would be duplicated events on the change stream, if the offset hadn't been sync'd and there was a invalidate event.